openedx/core/djangoapps/call_stack_manager/models.py

```diff
@@ -0,0 +1,6 @@
+"""
+Models file
+
+Note -
+Django tests searches for Model
+"""
```

Comment:
Yes. While creating Models for tests, `django` appears to search `models.py` . Local tests fail if we do not have `models.py`. Will try running tests on Jenkins if necessary.